### PR TITLE
Fixup links

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: templates/header.msi.html [hunk 1]
```diff
@@ -4,12 +4,6 @@
 
 <p>
   This is the MSI package repository of {{product_name}} for installation.
-
-  <pre class="text-white bg-dark">
```
Comment (oleg-nenashev) [inline templates/header.msi.html:8]: 👍

**Comment by oleg-nenashev** [comment]
Thanks for fixing it

**Comment by timja** [comment]
> Thanks for fixing it

np, can you please trigger the packaging job again?

**Comment by oleg-nenashev** [comment]
yes

**Pull request merged**